redbot/cogs/downloader/downloader.py

```diff
@@ -1405,14 +1362,50 @@ async def findcog(self, ctx: commands.Context, command_name: str) -> None:
             cog_name = self.cog_name_from_instance(cog)
             installed, cog_installable = await self.is_installed(cog_name)
             if installed:
-                msg = self.format_findcog_info(command_name, cog_installable)
-            else:
-                # Assume it's in a base cog
-                msg = self.format_findcog_info(command_name, cog)
+                from_git = True
+                made_by = ", ".join(cog_installable.author) or _("Missing from info.json")
+                repo_url = (
+                    _("Missing from installed repos")
+                    if cog_installable.repo is None
+                    else cog_installable.repo.clean_url
+                )
+                cog_name = cog_installable.name
+            elif cog.__module__.startswith("redbot."):  # core commands or core cog
+                from_git = False
+                made_by = "Cog Creators"
+                repo_url = "https://github.com/Cog-Creators/Red-DiscordBot"
+                cog_name = cog.__class__.__name__
+            else:  # assume not installed via downloader
+                from_git = False
+                made_by = "Unknown"
+                repo_url = "None - this cog wasn't installed via downloader"
+
         else:
             msg = _("This command is not provided by a cog.")
+            await ctx.send(msg)
+            return
 
-        await ctx.send(box(msg))
+        if await ctx.embed_requested():
+            embed = discord.Embed(color=(await ctx.embed_colour()))
+            embed.add_field(name="Command:", value="`%s`" % (command_name), inline=True)
+            embed.add_field(name="Cog Name:", value="`%s`" % (cog_name), inline=True)
+            embed.add_field(name="Made by:", value="`%s`" % (made_by), inline=True)
+            embed.add_field(name="Repo URL:", value=repo_url)
+            if from_git and cog_installable.repo is not None and cog_installable.repo.branch:
+                embed.add_field(
+                    name="Repo branch:", value="`%s`" % (cog_installable.repo.branch), inline=False
+                )
+            await ctx.send(embed=embed)
+
+        else:
+            msg = _(
+                "Command: {command}\nCog name: {cog}\nMade by: {author}\nRepo URL: {repo_url}\n"
```

Comment:
```suggestion
                _("Command: {command}\nCog name: {cog}\nMade by: {author}\nRepo URL: {repo_url}\n")
```

Reply:
It's already wrapped right in `_()` above this line.

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaa im officially dumber than slime.